python/cudf/cudf/tests/test_orc.py

```diff
@@ -477,3 +477,16 @@ def test_int_overflow(tmpdir):
     df.to_orc(file_path)
 
     assert_eq(cudf.read_orc(file_path), df)
+
+
+def test_row_group_indexing():
```

Comment:
This tests passes locally even without your changes :thinking:

Reply:
Yes it passes locally without my change as well. I think the test is meant to safeguard what was fixed in #6342

Reply:
Can you please verify that?

Reply:
I checked, with and without the change, it doesn't seems to fail even for very large number of rows, will try to get a failing test case.

Reply:
couldn't find a test case where it would fail without fix from PR #6342 , so removing this test.